src/migrations/index.js

```diff
@@ -183,6 +183,16 @@ const migrations = [
       console.log('added creator_id field to campaign')
     }
   },
+  {
+    auto: true, // 14
+    date: '2019-05-13',
+    migrate: async () => {
+      console.log('setting sequence value for canned_response')
+      const maxId = await r.knex('canned_response').max('id').first()
```

Comment:
I think we need to accomodate zero-canned responses case, so maybe `const maxId = (await ....) || 0`

Reply:
I tested that case @schuyler1d.  If there are no rows `await r.knex('canned_response').max('id').first()` returns 0.

Reply:
👍